animalfacts.py

```diff
@@ -1433,6 +1434,22 @@ def animalfactsbot(reddit):
     'Hamsters range in size from the largest breed, the European hamster at 13.4 inches (34 cm) long, to the smallest, the dwarf hamster at 2 - 4 inches (5.5 - 10.5 cm) long.',
     )
 
+HAWK_FACTS = (
+    'Hawks can vary in size depending on the species.',
+    'Hawks have excellent eyesight. They can see 8 times better than humans and can locate prey from 100 feet away.',
+    'Hawks are diurnal animals (active during the day.',
+    'Hawks build nests on trees or build nests on the ground in marshes.',
+    'Average lifespan of a hawk is 10 to 20 years in the wild.',
+    'Hawks are characterized by sharp talons, a curved bill, and muscular legs.',
+    'The largest species of hawks, the northern goshawk can weigh up to 2.2 kilograms (4.85 pounds)',
+    'Hawks eat small mammals such as mice, squirrels, and rabbits, as well as insects, smaller birds, turtles, and reptiles',
+    'After eating, a hawk will regurgitate a pellet that contains feathers and small bones',
+    'Hawks tend to mate during the spring and spend the majority of their time alone',
+    'Female hawks lay 1 to 5 eggs per year',
+    'Both the male and female hawks will create their nest, improve it, and take care for their eggs',
+    'Female hawks are generally larger than male hawks'
```

Comment:
Any chance of a comma here at the end to match the other blocks and to make it easier to add new lines?

Reply:
See 769bc04265ce52192628dfa19c5f39765a1c12ce